packages/dialtone-vue2/localization/localization.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta, Subtitle, Canvas, Story } from '@storybook/blocks';
+import * as LocalizationStories from './localization.stories.js';
+
+<Meta of={LocalizationStories} />
+
+# Dialtone-vue localization plugin
+
+<Subtitle>
+  Enables Dialtone-vue components localization.
+  It is very simple and powerful as it allows you to quickly switch between languages and requires minimal setup.
+  <br/>
+  ⚠️ You MUST initialize the localization plugin before using any Dialtone-vue component to avoid issues.
```

Comment:
Feels like we need more info here, how do you change the language? What does this affect? We probably want to state that this will only affect static strings rendered within dialtone, it will not affect strings passed in through props.